app/views/claim_mailer/approved.text.erb

```diff
@@ -2,14 +2,14 @@ Dear <%= @display_name %>,
 
 Your claim <%= @claim_description %> has been approved.
 
-We will send you another email when we have processed your payment, which may take up to <%= claim_decision_deadline_in_weeks %>.
+You will receive your payment within <%= claim_decision_deadline_in_weeks %>. We will send you another email when we make your payment.
```

Comment:
A _decision_ is made in `claim_decision_deadline_in_weeks` weeks, but it may take up to another 6 weeks (according to the changes made to the submission email template) to make a payment. The change to the code is correct, but I think this content is actually wrong. Might be worth checking so we can get it right before the next release.

Reply:
@jacksonj04 The actual copy I was given specifically states "6 weeks" and I substituted in `claim_decision_deadline_in_weeks` because that was used in the current live version of the copy, which may have been a mistake.

I'm assuming that the 'approved' email goes out at the point the decision has been made, and therefore claimants will be paid within 6 weeks of receipt of it?